router/src/main/resources/Dockerfile

```diff
@@ -13,11 +13,29 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 # -----------------------------------------------------------------------
-
 FROM envoyproxy/envoy-alpine:v1.17.1
+FROM frolvlad/alpine-glibc:alpine-3.12_glibc-2.31
```

Comment:
There are some later versions as well. Shall we give it a try with 
frolvlad/alpine-glibc:alpine-3.12_glibc-2.32

Reply:
Yes it also worked.